aztec/src/main/kotlin/org/wordpress/aztec/ime/EditorInfoUtils.kt

```diff
@@ -0,0 +1,55 @@
+package org.wordpress.aztec.ime
+
+import android.os.Build
+import android.view.inputmethod.EditorInfo
+import java.util.Arrays
+
+object EditorInfoUtils {
+    @JvmStatic
+    fun areEditorInfosTheSame(ed1: EditorInfo, ed2: EditorInfo): Boolean {
+        if (ed1 == ed2) {
```

Comment:
One thing that confuses me is why referential equality is not sufficient. Also, if we make a copy (`inputConnectionEditorInfo`) won't the referential comparison in `areEditorInfosTheSame` always be false?

Reply:
Good question! I thought the same initially (that comparing both EditInfo object rerferences should be enough), but later realized the object may be the same but still the `InputConnection` itself can change the attributes within `EditInfo` when instantiated.
 so we need to check the parameters rather than the object. The referential equality check was then kept there just for the case both parameters are `null` - we can make it more explicit though.

Reply:
> InputConnection itself can change the attributes within EditInfo 
Ah, as we are merely keeping a copy for later comparison. That makes sense.

Re: both params being `null`, I was thinking since `inputConnectionEditorInfo` is always set to `outAttrs` if `null`, this would never be reached, but given the semantics of the function name, I think it makes total sense to leave it as is. Thanks for explaining!

Reply:
Gotcha! Definitely agreed, I'd also like to leave the comparison there, given the purpose of the method. Thank you! 🙇